torchbenchmark/models/gat/install.py

```diff
@@ -0,0 +1,11 @@
+
+import subprocess
+import sys
+
+
+def pip_install_requirements():
+    subprocess.check_call([sys.executable, '-m', 'pip', 'install', '-q', '-r', 'requirements.txt', '-f', 'https://data.pyg.org/whl/torch-2.0.0+cpu.html'])
```

Comment:
Does this link contain the packages building against PyTorch nightly? If it only has packages on PyTorch stable, pip will uninstall the nightly packages and reinstall stable one, which is not what we want.